package.json

```diff
@@ -7,10 +7,11 @@
     "@emotion/cache": "11.11.0",
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
+    "@mui/base": "5.0.0-beta.12",
```

Comment:
Why was this added? Maybe we can remove it? Unless it was needed for the patch...

Reply:
It was added in this commit: 6114409. It seems to be needed for the patch, as the code won't compile without it.